src/EndOfYearBanner/index.tsx

```diff
@@ -36,26 +40,15 @@ const catchAndLogErrors = (description: string, fn: () => void): void => {
     }
 };
 
-const urlObject = new URL(
-    'https://www.theguardian.com/info/ng-interactive/2020/dec/21/the-guardian-in-2020',
-);
+const DEFAULT_IMAGE =
+    'https://i.guim.co.uk/img/media/c9ba78ef2b1a931aab5ca625ce49646e116b11b3/0_0_3200_1800/3200.png?quality=60&width=930&s=72ff133ea3e4516f5a353213e7a62e8a';
```

Comment:
I wonder if a default image at this level makes sense, I feel like it might be more confusing for marketing than just saying "you must provide an image". What do you think? If we did want a default an alternative could be to wrap `EndOfYearBanner` in a more specifically named component, which threads the image prop through (I think similar to the way `DigitalSubscriberAppBanner` is implemented on top of `AppBanner`).

Reply:
(This is a genuine question btw! I'm not sure how I feel about the default! Maybe Yusuf will have an opinion.)

Reply:
I've removed the default image as the imageUrl is now required by canRender and the story.